src/main/java/views/AcesUp/Choice.html

```diff
@@ -0,0 +1,28 @@
+<html>
+
+<link href="/assets/css/AcesUp.css" rel="stylesheet">
+<script type="text/javascript" src="/assets/webjars/jquery/2.1.3/jquery.js"></script>
+
+<body>
+<center><h1>Choose the Version</h1></center>
+
+<div class="centered">
```

Comment:
I thought you were just going to use the center html tag, but way to take the high road